pkg/workloads/console/integration/integration_test.go

```diff
@@ -188,15 +192,38 @@ var _ = Describe("Console", func() {
 			By("Expect job was created")
 			job := &batchv1.Job{}
 			identifier, _ := client.ObjectKeyFromObject(csl)
+			identifier.Name += "-console"
 			err := mgr.GetClient().Get(context.TODO(), identifier, job)
 
 			Expect(err).NotTo(HaveOccurred(), "failed to find associated Job for Console")
+
+			By("Expect job and pod labels are correctly populated from the console template and console")
+			Expect(
+				job.ObjectMeta.Labels["user"]).To(Equal("system-unsecured"))
```

Comment:
Missing a description parameter on the `To` function (to tell us the issue if this assertion fails)
e.g.
> job should have a user label matching the console owner

Applies to the expectation below also